src/constants.js

```diff
@@ -223,6 +223,7 @@ const FEATURE_FLAGS = [
   'goal_source',
   'merge_goals',
   'monitoring',
+  'training_reports',
```

Comment:
this shouldn't be in this PR, should it?

Reply:
I think this was because my db already expected it. Will remove.